syntax = "proto3";

import "base.proto";


enum TestEnum
{
	TestEnum_NONE=0;
	T1=1;
	T2=2;
}

message TestTable
{		
	uint64			id = 265 [(db.OptKey)=K_MAIN_KEY];	
	uint32			name_32 = 2 [(db.OptKey)=K_INDEX];			
	uint64			name_64 = 3;		
	bool			name_bool = 4;	
	string			name_str = 53[(db.OptKey)=K_INDEX];	
	bytes			name_bytes = 233;
	TestEnum		name_enum = 133;	
	uint64			id23 = 63;

}

message SubMsg
{
	TestEnum		name_enum = 133;	
	bool			name_bool = 4;	
}

message TTT3
{
	uint64 id = 14;	
	string name = 13[(db.OptKey)=K_MAIN_KEY];	
	uint32 t = 1;
	SubMsg sub_msg = 2;
	string t23 = 32;
}


